packages/docs-examples/components/notification-center/notification-center-empty/notification-center-empty-example.ts

```diff
@@ -0,0 +1,104 @@
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { AsyncPipe } from '@angular/common';
+import { ChangeDetectionStrategy, Component, computed, inject } from '@angular/core';
+import { takeUntilDestroyed, toSignal } from '@angular/core/rxjs-interop';
+import { KbqLuxonDateModule } from '@koobiq/angular-luxon-adapter/adapter';
+import { KbqBadgeModule } from '@koobiq/components/badge';
+import { KbqButtonModule, KbqButtonStyles } from '@koobiq/components/button';
+import { KbqComponentColors, KbqFormattersModule, PopUpPlacements, ThemeService } from '@koobiq/components/core';
+import { KbqDropdownModule } from '@koobiq/components/dropdown';
+import { KbqEmptyStateModule } from '@koobiq/components/empty-state';
+import { KbqIconModule } from '@koobiq/components/icon';
+import { KbqNavbarIcModule } from '@koobiq/components/navbar-ic';
+import { KbqNotificationCenterModule, KbqNotificationCenterService } from '@koobiq/components/notification-center';
+import { KbqTopBarModule } from '@koobiq/components/top-bar';
+import { of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+type ExampleAction = {
+    id: string;
+    icon?: string;
+    text?: string;
+    action?: () => void;
+    style: KbqButtonStyles | string;
+    color: KbqComponentColors;
+};
+
+enum NavbarIcItems {
+    Assets,
+    Issues,
+    Incidents,
+    Policies,
+    Security
+}
+
+/**
+ * @title notification-center-empty
+ */
+@Component({
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    standalone: true,
+    selector: 'notification-center-empty-example',
+    templateUrl: 'notification-center-empty-example.html',
+    imports: [
+        KbqNotificationCenterModule,
+        KbqNavbarIcModule,
+        KbqBadgeModule,
+        KbqIconModule,
+        KbqEmptyStateModule,
+        KbqTopBarModule,
+        KbqButtonModule,
+        KbqDropdownModule,
+        AsyncPipe,
+        KbqLuxonDateModule,
+        KbqFormattersModule
+    ],
+    providers: [
+        { provide: KbqNotificationCenterService, useClass: KbqNotificationCenterService }]
+})
+export class NotificationCenterEmptyExample {
+    protected readonly currentTheme = toSignal(
+        inject(ThemeService, { optional: true })?.current.pipe(
+            map((theme) => theme && theme.className.replace('kbq-', ''))
+        ) || of('light'),
+        { initialValue: 'light' }
+    );
+
+    protected readonly srcSet = computed(() => {
+        const currentTheme = this.currentTheme();
+
+        return `assets/images/${currentTheme}/empty_192.png 1x, assets/images/${currentTheme}/empty_192@2x.png 2x`;
```

Comment:
иначе не будет картинки на стакблитз
```suggestion
        return `https://koobiq.io/assets/images/${currentTheme}/empty_192.png 1x, assets/images/${currentTheme}/empty_192@2x.png 2x`;
```